README.md

```diff
@@ -34,41 +33,9 @@ The Simple Report application stack consists of:
-1. Install AdoptOpenJDK 11, either using [jabba](https://github.com/shyiko/jabba) or your preferred method. This is useful for IDE integration (IntelliJ, VS Code, etc.)
-1. Install yarn by running `npm install --global yarn`.
-
-Once those dependencies are installed, follow these steps to get the app up and running:
-
-1. Clone this repository: `git clone git@github.com:CDCgov/prime-simplereport.git`
-2. Run `yarn install` in the root of the repository to install pre-commit hooks using lefthook
-3. Run `cp .env.sample .env` in the root of the repository, then obtain the needed secrets from Azure or another developer. Be sure to set the `OKTA_API_KEY` environment variable. You can generate an API token for yourself by logging into the Okta Preview [admin panel](https://hhs-prime-admin.oktapreview.com) and going into Security > API > Tokens.
-4. Run `sudo vim /etc/hosts` (or use your editor of choice) and add a line to the bottom with the following contents and save: `127.0.0.1 localhost.simplereport.gov`
-5. Follow these steps from GitHub for [authenticating to the container registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-to-the-container-registry).
-6. Run `yarn start` in the root of the repository to start the app
-7. Make your user a support admin by assigning yourself the `SR-DEV-ADMINS` group in Okta Preview.
-8. When the frontend and backend builds are complete, visit https://localhost.simplereport.gov/app
-
-You should see the support admin page. The application does not yet start with any seed data, so you'll need to add your own organization and facilities. You can create your first org at https://localhost.simplereport.gov/app/sign-up.
-
-#### Notes
-Make sure docker has access to enough resources on your local machine. You'll know it doesn't if you get an error from any of your containers similar to this:
-```
-"The build failed because the process exited too early. This probably means the system ran out of memory or someone called `kill -9` on the process."
-```
-
-How to update resources limits on your [Mac](https://docs.docker.com/desktop/mac/#resources), and [Windows](https://docs.docker.com/desktop/windows/#resources) machines.
-
-## Running outside of docker
-
-If you wish to run outside of docker compose (bare metal), you'll need to install all the dependencies locally.
-See more [info here](https://github.com/CDCgov/prime-simplereport/wiki/Running-outside-of-docker).
+There are two different ways to run the app for local development
+1. [Docker and docker compose](https://github.com/CDCgov/prime-simplereport/wiki/%5BSetup%5D-Docker-and-docker-compose-development)
+2. [Bare metal (outside of Docker)](https://github.com/CDCgov/prime-simplereport/wiki/%5BSetup%5D-Running-outside-of-docker)
```

Comment:
Not really part of this PR, but I think this wiki page has a little too much data - it also includes information about rollbacks, database cleaning, etc that is already on this README. If you could break those sections into separate wiki pages it'd be great, thank you!

Reply:
Yes! It's on my todo list hahah Thank you for flagging this! 😸